scss/_roster.scss

```diff
@@ -233,7 +233,7 @@
     }
 }
 
-#jsxc_roster > {
+#jsxc_roster > & {
     &.jsxc_bottom {
```

Comment:
you talk about :
```
#jsxc_roster > {
     .jsxc_bottom {
```
?

that results in:
`Invalid parent selector` -> ` #jsxc_roster > { `

btw I am using:
```
node-sass 3.4.2
libsass 3.3.2
```